tests/layer_tests/tensorflow_tests/test_tf_Rint.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+class TestRint(CommonTFLayerTest):
+    def create_tf_rint_net(self, input_shape, input_type):
```

Comment:
@awayzjj, can you please customize input data generation for this test? I am not sure about common data generator. Here we can control locally and make sure that input data is not necessarily integral